pkg/server/server.go

```diff
@@ -307,3 +319,26 @@ func setNoProxyEnv(config *config.Control) error {
 	}, ",")
 	return os.Setenv("NO_PROXY", envList)
 }
+
+func writeConfigSymlink(kubeconfig, kubeconfigSymlink string) error {
+	if _, err := os.Stat(kubeconfigSymlink); err == nil {
+		if err := os.Remove(kubeconfigSymlink); err != nil {
```

Comment:
You can just call os.Remove and then ignore os.IsNotExists(err).  If you don't do that you have to change the os.Stat to Lstat.  Because stat will return not found err if the symlink exists but what it points to is missing.  Lstat will stat the symlink file directly.